src/content/learn/installation.md

```diff
@@ -1,25 +1,25 @@
 ---
-title: Installation
+title: Instalace
 ---
 
 <Intro>
 
-React has been designed from the start for gradual adoption. You can use as little or as much React as you need. Whether you want to get a taste of React, add some interactivity to an HTML page, or start a complex React-powered app, this section will help you get started.
+React byl od začátku navržen pro postupnou integraci. Můžete používat tolik Reactu, kolik potřebujete. Ať už chcete získat představu o Reactu, přidat nějakou interaktivitu do HTML stránky, nebo začít komplexní aplikaci poháněnou Reactem, tato sekce vám pomůže začít.
 
 </Intro>
 
 <YouWillLearn isChapter={true}>
 
-* [How to start a new React project](/learn/start-a-new-react-project)
-* [How to add React to an existing project](/learn/add-react-to-an-existing-project)
-* [How to set up your editor](/learn/editor-setup)
-* [How to install React Developer Tools](/learn/react-developer-tools)
+* [Jak začít nový projekt v Reactu](/learn/start-a-new-react-project)
+* [Jak přidat React do existujícího projektu](/learn/add-react-to-an-existing-project)
+* [Jak nastavit váš editor](/learn/editor-setup)
+* [Jak instalovat nástroje pro React vývojáře](/learn/react-developer-tools)
 
 </YouWillLearn>
 
-## Try React {/*try-react*/}
+## Vyzkoušejte React {/*try-react*/}
 
-You don't need to install anything to play with React. Try editing this sandbox!
+Není třeba nic instalovat, abyste si mohli React vyzkoušet. Zkuste třeba upravit tento sandbox přímo v prohlížeči!
```

Comment:
I am open for a discussion. 
```suggestion
Není třeba nic instalovat, abyste si mohli React vyzkoušet. Zkuste třeba upravit toto pískoviště přímo v prohlížeči!
```

Reply:
Hmm, let's talk about it on sync. Not sure if I've ever seen a translation of "sandbox" on czech. I view "sandbox" as anglicism that already has a place in our vocab thanks to Windows Sandbox, but I might be wrong 🤔 ...